src/OneBlinkFormBase.tsx

```diff
@@ -139,6 +141,11 @@ export type OneBlinkFormBaseProps = OneBlinkReadOnlyFormProps & {
    * for the structure of the argument and a sample function to be used.
    */
   onUploadAttachment?: typeof attachmentsService.uploadAttachment
+  /**
+   * Determines whether to use checkboxes or invisible recaptcha v2 for captcha
```

Comment:
Can we please add a something to this doc that indicates the default if leave as `undefined`